VK/VK/MyGroupsScreen/MyGroupsTableViewController.swift

```diff
@@ -77,6 +87,22 @@ final class MyGroupsTableViewController: UITableViewController {
 
     // MARK: Private methods
 
+    private func fetchMyGroupes() {
+        NetworkService.shared.fetchClientsGroups { groups in
+            let array = groups.response.items.map { group in
+                NetworkUnit(
```

Comment:
```suggestion
                NetworkUnit(
```
лучше в NetworkUnit добавь в синит тип группы а тут в 1 строчку переделай $0